assets/js/get_list.js

```diff
@@ -127,9 +159,26 @@ async function getUnacceptedList() {
     attendee.querySelector(".g-name").appendChild(document.createTextNode(user.guardian_name ? user.guardian_name : ""));
     attendee.querySelector(".g-email").appendChild(document.createTextNode(user.guardian_email ? user.guardian_email : ""));
     attendee.querySelector(".g-tel").appendChild(document.createTextNode(user.guardian_phone_number ? user.guardian_phone_number : ""));
-    attendee.querySelector(".github-user").appendChild(document.createTextNode(user.github_username ? user.github_username : ""));
-    attendee.querySelector(".linkedin-prof").appendChild(document.createTextNode(user.linkedin_profile ? user.linkedin_profile : ""));
+
+    if (user.github_username) {
+      var github_link = document.createElement("a");
+      github_link.href = user.github_username;
```

Comment:
User might provide just the username and not a url, right?

Reply:
kendall's signup form doesn't allow anything but an url